crates/router/src/compatibility/stripe/setup_intents/types.rs

```diff
@@ -243,9 +226,11 @@ impl TryFrom<StripeSetupIntentRequest> for payments::PaymentsRequest {
             client_secret: item.client_secret.map(|s| s.peek().clone()),
             setup_future_usage: item.setup_future_usage,
             merchant_connector_details: item.merchant_connector_details,
-            authentication_type,
```

Comment:
Why is authentication type removed from here ?

Reply:
Added it back. Thanks for pointing it out!